pkg/components/components.go

```diff
@@ -1036,9 +1036,15 @@ func GetCrd() *extv1.CustomResourceDefinition {
 
 	cipherSuites := func() []extv1.JSON {
 		suites := []extv1.JSON{}
+		m := make(map[string]bool)
 		for _, p := range tlsProfiles(ocpv1.TLSProfiles).sortedKeys() {
 			for _, c := range ocpv1.TLSProfiles[p].Ciphers {
-				suites = append(suites, extv1.JSON{Raw: []byte(fmt.Sprintf("\"%s\"", c))})
+				format := fmt.Sprintf("\"%s\"", c)
+				if m[format] {
+					continue
+				}
+				m[format] = true
+				suites = append(suites, extv1.JSON{Raw: []byte(format)})
```

Comment:
Do the json "encoding" at the end we don't need to store double quotes at the set

```suggestion
				if m[c] {
					continue
				}
				suites = append(suites, extv1.JSON{Raw: []byte(fmt.Sprintf(`"%s"`, c))})
```